docker/docker-compose.2204.59.yaml

```diff
@@ -38,6 +38,10 @@ services:
       - MAX_ALLOCS_ALLOWED_client_server_request_response_inline=244050
       - MAX_ALLOCS_ALLOWED_client_server_request_response_many=1198050
       - MAX_ALLOCS_ALLOWED_client_server_request_response_many_inline=889050
+      - MAX_ALLOCS_ALLOWED_client_server_request_response_no_promise_on_channel_creation=253050
+      - MAX_ALLOCS_ALLOWED_client_server_request_response_no_promise_on_channel_creation_inline=251050
+      - MAX_ALLOCS_ALLOWED_client_server_request_response_no_promise_on_channel_creation_many=1198050
+      - MAX_ALLOCS_ALLOWED_client_server_request_response_no_promise_on_channel_creation_many_inline=896050
```

Comment:
Rather than duping all of these tests wouldn't it make more sense to have a variant of one of 'create_client_stream_channel_*'?

Reply:
Yeah you're right that makes more sense. Just changed it.